src/parser/jobs/mnk/modules/PerfectBalance.tsx

```diff
@@ -303,5 +302,23 @@ export class PerfectBalance extends Gauge {
 				<Plural value={badActions} one="# use of" other="# uses of"/> uses of <DataLink action="FORM_SHIFT"/> or <DataLink action="ANATMAN"/> were used during <DataLink status="PERFECT_BALANCE"/>.
 			</Trans>,
 		}))
+
+		// Future TODO: Calculate whether the overcap mattered
+		const nadiOvercap = this.nadiGauge.overcap
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.PHANTOM_RUSH.icon,
+			content: <Trans id="mnk.pb.suggestions.nadi-overcap.content">
+				Generating a Lunar or Solar Nadi while already in possession of that Nadi means fewer uses of <DataLink action="PHANTOM_RUSH" /> over the course of the fight. Try not to overcap either of your Nadis.
+			</Trans>,
+			tiers: {
+				2: SEVERITY.MINOR, // Start at 2 since 1 might be expected depending on opener
+				3: SEVERITY.MEDIUM,
+				4: SEVERITY.MAJOR,
+			},
```

Comment:
This severity seems too low.  Losing a Phantom Rush would be more potency loss than overwriting a Polyglot (and more potency loss than using 5 Celestial Revolutions to avoid doubling up), but we score both of those other mistakes as always MAJOR.  If we need to forgive one in the opener, can we implement a start of fight forgiveness for overcap and then otherwise flag any overcap as bad?

See this log for one where the overcap definitely lost a usage - looks like went for an opener with double Solar but then did two more PBs before getting to Phantom Rush http://localhost:3000/fflogs/GjQ1RAVaD4cWk6Xv/1/49

Reply:
I'm ok with changing 2 to Medium and 3 to Major for this PR. Trying to add a fight timing forgivenness factor and/or the noted "figure out if the overcap actually lost them a Phantom Rush usage" changes I think can wait for another PR and/or a 7.0 change.

Reply:
Changed to MAJOR at overcap >=2 based on Discord discussion